.gitignore

```diff
@@ -123,15 +123,15 @@ generated/
 #miscell
 .vscode/
 
-!screenshots/README.md
-screenshots/
+# !screenshots/README.md
+# screenshots/
 
 .firebase/
 .firebaserc
 
 static/firebase.config.json
 
-# vi/vim
```

Comment:
I don't use vim anymore but I'd suggest leaving these in gitignore if you don't mind